cfme/infrastructure/virtual_machines.py

```diff
@@ -366,7 +366,8 @@ class InfraVmSnapshotAddView(InfraVmView):
     @property
     def is_displayed(self):
         """Is this view being displayed"""
-        return False
+        expected_title = "Adding a new Snapshot"
```

Comment:
:+1: 
Are you confirmed its not version dependent?

Reply:
Yes, it's the same in 5.9 and 5.10.

Reply:
you can also make is_displayed raise NotImplementedError instead